cephfs/file.go

```diff
@@ -391,3 +391,25 @@ func (f *File) Fsync(sync SyncChoice) error {
 func (f *File) Sync() error {
 	return f.Fsync(SyncAll)
 }
+
+// Truncate sets the size of the open file.
+// NOTE: In some versions of ceph a bug exists where calling ftruncate on a
+// file open for read-only is permitted. The go-ceph wrapper does no additional
+// checking and will inherit the issue on affected versions of ceph.  Please
+// refer to the following issue for details:
+// https://tracker.ceph.com/issues/48202
+//
+// Implements:
+//  int ceph_ftruncate(struct ceph_mount_info *cmount, int fd, int64_t size);
+func (f *File) Truncate(size int64) error {
```

Comment:
Any reason for not naming it Ftruncate, that's the nomenclature existing functions in the file have followed?

Reply:
When there's an established syscall interface with equivalent functionality (no alternative args, etc) that's part of Golang's file interface we've been using those names. This is such a case.